### PR TITLE
277 model list display

### PR DESCRIPTION
**Pull request by Connor-Moesol**
Feature: ModelListDisplay component that displays the array of values as a comma separated list. 
Accepts a render fn prop that maps out how each item should be displayed.

### CODE REVIEW

File: src/BasicDisplays/ModelListDisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+export interface ModelListDisplayProps extends FormDisplayChildProps {
+  className?: string;
+  getDisplayValue?: (value: FormDisplayChildProps['value']) => ReactNode;
+}
+
+export const ModelListDisplay = ({
+  value,
+  getDisplayValue = (val) => {
+    return val?._display_value ?? 'None';
```
Comment (robxbob) [inline src/BasicDisplays/ModelListDisplay.tsx:14]: use camelCasing for displayValue

**Comment by robxbob** [comment]
Just for extra clarity,

The ModelListDisplay (when used correctly) will be passed with a value of type [{...}] where the object is the model object that will/should have the following fields: id, displayValue. 

The `getDisplayValue` prop returns a `ReactNode` because it is meant to allow customization on each item in the array. Add another storybook story with ` getDisplayValue: (value) => <span key={value} className="bg-red-200">{value}</span>` to understand how it could/would be used.

File: src/BasicDisplays/ModelListDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  className,
+  ...props
+}: ModelListDisplayProps) => {
+  const modelList = value
+    .map((val: ReactNode) => getDisplayValue(val))
+    .join(', ');
```
Comment (robxbob) [inline src/BasicDisplays/ModelListDisplay.tsx]: Assumes getDisplayValue returns a string, which it doesn't; it returns a `ReactNode`, so `.join` shouldn't be possible.

Comment (robxbob) [reply src/BasicDisplays/ModelListDisplay.tsx]: Also your `.map` can just be passed the `getDisplayValue` fn for simplicity

File: src/BasicDisplays/ModelListDisplay.tsx (hunk 1 above)
Comment (Connor-Moesol) [reply src/BasicDisplays/ModelListDisplay.tsx:14]: It looks like other places use `_display_value` is this changing? 
i.e. `ModelItemInput`

File: src/BasicDisplays/ModelListDisplay.tsx (hunk 2 above)
Comment (Connor-Moesol) [reply src/BasicDisplays/ModelListDisplay.tsx]: This sounds good, made some adjustments.

File: src/BasicDisplays/ModelListDisplay.tsx (hunk 1 above)
Comment (robxbob) [reply src/BasicDisplays/ModelListDisplay.tsx:14]: Yea, change all instances

**Pull request merged**